src/main/java/net/neoforged/neoforge/items/wrapper/RecipeWrapper.java

```diff
@@ -5,12 +5,11 @@
 
 package net.neoforged.neoforge.items.wrapper;
 
-import net.minecraft.world.Container;
-import net.minecraft.world.entity.player.Player;
 import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.crafting.RecipeInput;
 import net.neoforged.neoforge.items.IItemHandlerModifiable;
 
-public class RecipeWrapper implements Container {
+public class RecipeWrapper implements RecipeInput {
     protected final IItemHandlerModifiable inv;
```

Comment:
This could be a plain `IItemHandler`, right?

Reply:
Good call. Yeah that is valid to do.